### PR TITLE
[syvdk] Add extractor

### PR DESCRIPTION
**Pull request by misaelaguayo**
<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [ x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x ] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [ x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [ x] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [ ] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

### Description of your *pull request* and other information

</details>

DESCRIPTION

Fixes #

### CODE REVIEW

File: yt_dlp/extractor/syvdk.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        _NEXT_DATA = self._html_search_regex(
+            r'<script id="__NEXT_DATA__" type="application\/json">(?P<meta_dict>{.+})<\/script>', webpage,
+            'audio URL', group='meta_dict')
+        info = json.loads(_NEXT_DATA)
```
Comment (Ashish0804) [inline yt_dlp/extractor/syvdk.py:25]: use `self._search_nextjs_data()`

File: yt_dlp/extractor/syvdk.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        _id = str(info["props"]["pageProps"]["episodeDetails"][0]["id"])
+
+        return {
+            'id': _id,
+            'title': title,
+            'description': description,
+            'vcodec': 'none',
+            'ext': 'mp3',
+            'url': audio_url
+        }
```
Comment (Ashish0804) [inline yt_dlp/extractor/syvdk.py:39]:
```suggestion
        info_data = self._search_nextjs_data(webpage, video_id)["props"]["pageProps"]["episodeDetails"][0]

        return {
            'id': str(info['id']),
            'display_id': video_id,
            'title': try_get(info, lambda x: x['title']['rendered']),
            'description': try_get(info, lambda x: x['details']['post_title']),
            'vcodec': 'none',
            'ext': 'mp3',
            'url': info['details']['enclosure']
        }
```

**Comment by HobbyistDev** [comment]
Fixes #4077

File: yt_dlp/extractor/syvdk.py [hunk 3]
```diff
@@ -10,9 +10,10 @@ class SYVDKIE(InfoExtractor):
...
-            'description': 'Tidligere formand for Kristendemokraterne Isabella Arendt stiller op til folketingsvalget for Det Konservative Folkeparti. Men hvorfor netop de konservative? og hvordan harmonisere det med hendes ønske om en mere human udlændinge- og integrationspolitikken. Hun er med fra start og fortæller hvorfor. <br /><br />Vi har også LOKK - Landsorganisation af Kvindekrisecentre’s formand Karin Gaardsted i programmet. Vi får talt om handleplanen mod partnerdrab. Og om der behov for at ændre den generelle samtale om vold i nære relationer og kvindedrab.'
+            'description': 'Tidligere formand for Kristendemokraterne Isabella Arendt stiller op til folketingsvalget for Det Konservative Folkeparti. Men hvorfor netop de konservative? og hvordan harmonisere det med hendes ønske om en mere human udlændinge- og integrationspolitikken.  Hun er med fra start og fortæller hvorfor. <br /><br />Vi har også LOKK - Landsorganisation af Kvindekrisecentre’s formand Karin Gaardsted i programmet. Vi får talt om handleplanen mod partnerdrab. Og om der behov for at ændre den generelle samtale om vold i nære relationer og kvindedrab.'
```
Comment (HobbyistDev) [inline yt_dlp/extractor/syvdk.py:16]: I think this line too long. Please use md5 instead

Comment (misaelaguayo) [reply yt_dlp/extractor/syvdk.py:16]: First pr, so not sure what that usually looks like. Replace the string with its md5 equivalent?

Comment (pukkandan) [reply yt_dlp/extractor/syvdk.py:16]:
```suggestion
            'description': 'md5:abcd'
```

replace `abcd` with the md5

Comment (HobbyistDev) [reply yt_dlp/extractor/syvdk.py:16]: > First pr, so not sure what that usually looks like. Replace the string with its md5 equivalent?

Yes, it must looks like
```
'description': 'md5:<your_md5_here>'
```
If you didn't have hash checker, you can get the md5 from test, just set `md5:fixme` or another value and then the test give you right md5 value

**Review by Ashish0804 — APPROVED** [review]
lgtm

**Comment by misaelaguayo** [comment]
@pukkandan could you merge this pr?

File: yt_dlp/extractor/syvdk.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            'ext': 'mp3',
+            'title': 'Isabella Arendt stiller op for De Konservative',
+            'description': 'md5:f5fa6a431813bf37284f3412ad7c6c06'
+        }
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        info_data = self._search_nextjs_data(webpage, video_id)["props"]["pageProps"]["episodeDetails"][0]
```
Comment (pukkandan) [inline yt_dlp/extractor/syvdk.py:23]:
```suggestion
        info_data = self._search_nextjs_data(webpage, video_id)['props']['pageProps']['episodeDetails'][0]
```

File: yt_dlp/extractor/syvdk.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+from .common import InfoExtractor
+from ..utils import try_get
+
+
+class SYVDKIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?24syv\.dk/episode/(?P<id>[a-zA-Z0-9-]+)'
```
Comment (pukkandan) [inline yt_dlp/extractor/syvdk.py:6]:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?24syv\.dk/episode/(?P<id>[\w-]+)'
```

File: yt_dlp/extractor/syvdk.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        info_data = self._search_nextjs_data(webpage, video_id)["props"]["pageProps"]["episodeDetails"][0]
+
+        return {
+            'id': str(info_data['id']),
+            'display_id': video_id,
+            'title': try_get(info_data, lambda x: x['title']['rendered']),
+            'description': try_get(info_data, lambda x: x['details']['post_title']),
```
Comment (pukkandan) [inline yt_dlp/extractor/syvdk.py:29]: Use traverse_obj

File: yt_dlp/extractor/syvdk.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+            'vcodec': 'none',
+            'ext': 'mp3',
+            'url': info_data['details']['enclosure']
+        }
```
Comment (pukkandan) [inline yt_dlp/extractor/syvdk.py:33]:
```suggestion
        return {
            'id': str(info_data['id']),
            'vcodec': 'none',
            'ext': 'mp3',
            'url': info_data['details']['enclosure'],
            'display_id': video_id,
            'title': traverse_obj(info_data, ('title', 'rendered')),
            'description': traverse_obj(info_data, ('details', 'post_title')),
        }
```

File: yt_dlp/extractor/syvdk.py [hunk 8]
```diff
@@ -0,0 +1,33 @@
+from .common import InfoExtractor
+from ..utils import try_get
```
Comment (pukkandan) [inline yt_dlp/extractor/syvdk.py:2]:
```suggestion
from ..utils import traverse_obj
```

**Review by pukkandan — APPROVED** [review]

**Pull request merged**